### DISCUSSION TITLE
Bug: macOS: background bleeds when resizing splits

### DISCUSSION THREAD

**Discussion by kdrag0n** [ghostty-org/ghostty#4182] [Feature Requests, Ideas] [answered]
https://github.com/user-attachments/assets/d596061c-f638-4850-8ea3-b5ecc78f813b

Possibly the same bug as #4180.

**Comment by mitchellh** [comment, marked as answer]
This is a dupe. Please search but I was just talking about this with someone yesterday.

**Reply by kdrag0n** [reply]
I couldn't find anything when searching for flicker / background / resize before opening this. Do you have a link?

**Reply by hit0ri** [reply]
#3508 might be the one, although it is more similar to #4181

**Reply by hit0ri** [reply]
there's also a relevant discord help topic [here](https://discord.com/channels/1005603569187160125/1318616867576746065)
